### PR TITLE
Improve calculate go enrichment

### PR DESCRIPTION
**Pull request by jpquast**

### CODE REVIEW

File: R/calculate_go_enrichment.R [hunk 1]
```diff
@@ -438,6 +446,11 @@ if you used the right organism ID.", prefix = "\n", initial = ""))
     dplyr::ungroup() %>%
     dplyr::filter(.data$n_detected_proteins_in_process >= min_n_detected_proteins_in_process)
 
+  if (replace_long_name & !missing(go_annotations_uniprot)) {
+    filtered_result_table <- filtered_result_table %>%
+      mutate(term = ifelse(nchar(.data$term) > 50, .data$go_id, .data$term))
```
Comment (elena-krismer) [inline R/calculate_go_enrichment.R:451]: dplyr::mutate

File: R/calculate_go_enrichment.R [hunk 2]
```diff
@@ -464,6 +479,10 @@ if you used the right organism ID.", prefix = "\n", initial = ""))
       dplyr::filter(!!rlang::ensym(type) <= threshold)
   }
 
+  # move label if bar is less than 20% (default) of largest bar
+  plot_input <- plot_input %>%
+    mutate(hjust = ifelse((.data$neg_log_sig / max(.data$neg_log_sig)) < label_move_frac, -0.15, 1.05))
```
Comment (elena-krismer) [inline R/calculate_go_enrichment.R:484]: dplyr::mutate

**Review by elena-krismer — APPROVED** [review]
just add the dplyr:: for consistency else everything is perfect :)

File: R/try_query.R [hunk 3]
```diff
@@ -34,12 +38,18 @@ try_query <-
...
     while (!is(query_result, "response") &
       try_n < max_tries &
+      # this ifelse stops requery if they timeout except for if try_if_timeout is TRUE
       !ifelse(is(query_result, "character"),
-        stringr::str_detect(query_result, pattern = "Timeout was reached"),
+        stringr::str_detect(query_result, pattern = "Timeout was reached") & !try_if_timeout,
         FALSE
-      )) { # this ifelse stops requery if the timeout is too low.
+      )
+      ) {
```
Comment (elena-krismer) [inline R/try_query.R]: i would add more verbous error messages like https://github.com/jpquast/protti/blob/fetch_uniprot_252/R/try_query.R line 59-70

**Review by elena-krismer — COMMENTED** [review]
i would consider adding a retry_delay as in PR #253 and making the error messages more clear. else everything is fine

File: R/try_query.R (hunk 3 above)
Comment (jpquast) [reply R/try_query.R]: added this.

**Pull request merged**